From db10415448158779127ad529335e2c447c5767c1 Mon Sep 17 00:00:00 2001
Message-Id: <db10415448158779127ad529335e2c447c5767c1.1644479477.git.geliang.tang@suse.com>
From: Ye Guojin <ye.guojin@zte.com.cn>
Date: Fri, 10 Dec 2021 07:14:24 +0000
Subject: [PATCH] selftests: mptcp: remove duplicate include in mptcp_inq.c
Git-commit: db10415448158779127ad529335e2c447c5767c1
Patch-mainline: v5.17-rc1
References: bsn#2
Modified-by-SEL: No


'sys/ioctl.h' included in 'mptcp_inq.c' is duplicated.

Reported-by: ZealRobot <zealci@zte.com.cn>
Signed-off-by: Ye Guojin <ye.guojin@zte.com.cn>
Reviewed-by: Matthieu Baerts <matthieu.baerts@tessares.net>
Link: https://lore.kernel.org/r/20211210071424.425773-1-ye.guojin@zte.com.cn
Signed-off-by: Jakub Kicinski <kuba@kernel.org>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 tools/testing/selftests/net/mptcp/mptcp_inq.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tools/testing/selftests/net/mptcp/mptcp_inq.c b/tools/testing/selftests/net/mptcp/mptcp_inq.c
index b8debd4fb5ed..29f75e2a1116 100644
--- a/tools/testing/selftests/net/mptcp/mptcp_inq.c
+++ b/tools/testing/selftests/net/mptcp/mptcp_inq.c
@@ -17,7 +17,6 @@
 #include <unistd.h>
 #include <time.h>
 
-#include <sys/ioctl.h>
 #include <sys/ioctl.h>
 #include <sys/socket.h>
 #include <sys/types.h>
-- 
2.34.1

